### PR TITLE
Split results.json

### PR DESCRIPTION
**Pull request by coolo**
This is not to be merged before openQA worker is adapted to this, but for review and discussion I open the pull request early

### CODE REVIEW

File: backend/baseclass.pm [hunk 1]
```diff
@@ -324,18 +324,9 @@ sub enqueue_screenshot() {
...
         $bmwqemu::screenshotQueue->enqueue($filename);
         $lastscreenshot          = $image;
         $lastscreenshotName      = $filename;
-        unless(symlink(basename($filename), $bmwqemu::liveresultpath.'/tmp.png')) {
-            # try to unlink file and try again
-            unlink($bmwqemu::liveresultpath.'/tmp.png');
-            symlink(basename($filename), $bmwqemu::liveresultpath.'/tmp.png');
-        }
-        rename($bmwqemu::liveresultpath.'/tmp.png', $bmwqemu::liveresultpath.'/last.png');
-
```
Comment (aaannz) [inline backend/baseclass.pm]: Is this WIP or how the livestream will work?

Comment (coolo) [inline backend/baseclass.pm]: the worker will push $last to the webui together with the usual status updates. I'll may add the filename of the last screenshot to the status.json

Comment (coolo) [inline backend/baseclass.pm]: even the last changed screenshot and the worker then can avoid uploading unchanged images.

Comment (coolo) [inline backend/baseclass.pm]: but yes, this is WIP - as long as I didn't finish the openQA side I expect that I have to touch this.

**Comment by coolo** [comment]
F**** - I played with git push options and managed to push to master ;(

**Comment by coolo** [comment]
I wasn't able to reopen, so this is now #199 - after force pushing the old master to master

**Comment by aaannz** [comment]
Rewrite the history, that's how winners rule :D

File: backend/baseclass.pm (hunk 1 above)
Comment (coolo) [inline backend/baseclass.pm]: So this won't work as the status is not written every time - so I went back to a symlink. But the symlink is in qemuscreenshots (where I actually missed it for a while when debugging ipmi :)

**Pull request merged**